luarules/gadgets/unit_shield_behaviour.lua

```diff
@@ -169,7 +169,7 @@ local function shieldNegatesDamageCheck(unitID, unitTeam, attackerID, attackerTe
 			return true
 		end
 
-		for subKey in pairs(shieldedUnits[unitID]) do
+		for shieldUnitID, _ in pairs(shieldedUnits[unitID]) do
 			if shieldedUnits[attackerID][subKey] then
```

Comment:
`subKey` -> `shieldUnitID` In VS Code, place your cursor in the variable you want to rename and hit F2. This will bring up the renaming tool, which will automatically rename all instances of the variable.

### EXTREME WARNING: It renames _all_ instances, and if it's a global... 😉

Reply:
I'm on baby watch, but, sounds like a good change. Thanks WtF